homeassistant/components/cloud/__init__.py

```diff
@@ -383,6 +403,8 @@ async def remote_prefs_updated(prefs: CloudPreferences) -> None:
         """Update remote status."""
         nonlocal cur_pref
 
+        async_dispatcher_send(hass, SIGNAL_CLOUDHOOKS_UPDATED, prefs.cloudhooks)
```

Comment:
From the event name, I expect that the cloudhooks are changed, but you are firing it on any cloud pref change